site/docs/voice/machineDetectionGuide.mdx

```diff
@@ -0,0 +1,60 @@
+---
+id: machineDetectionGuide
+title: Machine Detection
+slug: /voice/guides/machineDetection
+description: How to using Bandwidth's Voice API Answering Machine Detection Feature
+keywords:
+  - bandwidth
+  - voice
+  - answering
+  - machine
+  - detection
+hide_title: false
+image: ../../static/img/bandwidth-logo.png
+---
+
+This guide will show you how to use machine detection, using the `machineDetection` parameter in a [call creation request](/apis/voice#operation/createCall).
+
+You can use either a sync or an async mode for this operation.
```

Comment:
```suggestion
You can choose to use either synchronous (sync) or asynchronous (async) mode for this operation.
```

"a sync" and "an async" seems potentially confusing on a screen reader or just to someone scanning the documentation quickly